Cargo.toml

```diff
@@ -37,6 +37,11 @@ tokio-serial = { version = "5.4.4", optional = true, default-features = false }
 tokio-util = { version = "0.7.7", default-features = false, features = [
     "codec",
 ] }
+rustls-pemfile = {version="1.0.2", optional = true}
```

Comment:
These dependencies are nedeed only for the exampke.

Reply:
Ok. All suggestions accepted.